### PR TITLE
[MI-758] Display single order

### PR DESCRIPTION
**Pull request by joseconsador**
/cc @zendesk/mintegrations

### Description

Display single order when the order ID field has a value.

screen shot 2016-08-04 at 5 38 19 pm

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-758

### Risks

### CODE REVIEW

File: src/javascripts/shopify_app.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ var App = {
   resources: {
     PROFILE_URI       : '/admin/customers/search.json?query=email:',
     CUSTOMER_URI      : '%1$s/admin/customers/%2$d',
-    ORDERS_URI        : '%1$s/admin/orders.json?customer_id=%2$d&status=any&fields=name,id,currency%3$s',
+    ORDERS_URI        : '%1$s/admin/orders.json?limit=3,customer_id=%2$d&status=any&fields=name,id,currency%3$s',
```
Comment (miogalang) [inline src/javascripts/shopify_app.js]: Do we always get 3 orders? should this be configurable?

File: src/javascripts/shopify_app.js [hunk 2]
```diff
@@ -120,9 +120,46 @@ var App = {
...
+
+    this.zafClient.get('requirement:shopify_order_id').then(function(data) {
+      var fieldId = data['requirement:shopify_order_id'].requirement_id;
+      var fieldName = 'ticket.customField:custom_field_' + fieldId;
+
+      _self.zafClient.get(fieldName).then(function(customField) {
+        if (!_.isEmpty(customField[fieldName])) {
+          _self.ajax('getOrder', customField[fieldName]);
+        } else {
+          // Get customers's 50 most recent orders
```
Comment (miogalang) [inline src/javascripts/shopify_app.js]: let's just remove the comment?

**Comment by joseconsador** [comment]
@miogalang 💇

**Comment by samgavinio** [comment]
👍

**Comment by joseconsador** [comment]
@miogalang 💇

File: src/javascripts/shopify_app.js [hunk 3]
```diff
@@ -23,28 +23,44 @@ var App = {
   storeUrl: '',
 
   resources: {
-    PROFILE_URI       : '/admin/customers/search.json?query=email:',
-    CUSTOMER_URI      : '%1$s/admin/customers/%2$d',
-    ORDERS_URI        : '%1$s/admin/orders.json?customer_id=%2$d&status=any&fields=name,id,currency%3$s',
-    ORDER_PATH        : '%1$s/admin/orders/%2$d'
+    PROFILE_URI       : '/admin/customers/search.json',
```
Comment (miogalang) [inline src/javascripts/shopify_app.js]: nice

**Comment by miogalang** [comment]
👍

**Pull request merged**